pyam/core.py

```diff
@@ -72,8 +72,8 @@ class IamDataFrame(object):
 
     Parameters
     ----------
-    data : :class:`pandas.DataFrame`, :class:`ixmp.Scenario`,
-            or file-like object as str or :class:`pathlib.Path`
+    data : :class:`pandas.DataFrame`, :class:`ixmp.Scenario`,\
```

Comment:
I tried that, stickler doesn't like it because the line is too long... And the `\` follows the numpydoc convention, see https://github.com/numpy/numpydoc/issues/87